spyder/widgets/fileswitcher.py

```diff
@@ -706,7 +706,15 @@ def setup_file_list(self, filter_text, current_path):
         for result in sorted(results):
             index = result[1]
             path = paths[index]
-            icon = ima.get_icon_by_extension(path)
+            if sys.platform == 'darwin':
+                scalefactor = 0.9
```

Comment:
`scalefactor` -> `scale_factor`

Reply:
@juanis2112, you forgot to address this comment.

Reply:
@ccordoba12 right! Done in the new PR.

Reply:
Thanks @juanis2112!